client/post-editor/media-modal/index.jsx

```diff
@@ -263,9 +262,8 @@ export class EditorMediaModal extends Component {
 			this.setNextAvailableDetailView();
 		}
 
-		MediaActions.delete( site.ID, toDelete );
+		this.props.deleteMedia( site.ID, toDelete );
 		mcBumpStat( 'editor_media_actions', 'delete_media' );
-		this.props.deleteMedia( site.ID, map( toDelete, 'ID' ) );
```

Comment:
This action is now being dispatched as part of the data layer success handler `deleteMediaSuccess` created [here](https://github.com/Automattic/wp-calypso/pull/43618/files#diff-8a834454d3ca55002e933663ede8d986). This is also the duplicate I mentioned in the PR description.